scripts/ironic-common.sh

```diff
@@ -44,9 +66,12 @@ export LISTEN_ALL_INTERFACES="${LISTEN_ALL_INTERFACES:-true}"
 # Wait for the interface or IP to be up, sets $IRONIC_IP
 wait_for_interface_or_ip()
 {
-    # If $PROVISIONING_IP is specified, then we wait for that to become available on an interface, otherwise we look at $PROVISIONING_INTERFACE for an IP
+    # If $PROVISIONING_IP is specified, then we wait for that to become
+    # available on an interface, otherwise we look at $PROVISIONING_INTERFACE
+    # for an IP
     if [[ -n "$PROVISIONING_IP" ]]; then
```

Comment:
```suggestion
    if [[ -n "${PROVISIONING_IP}" ]]; then
```
While touching basically all lines around it, might fix it while at it.

Reply:
done